src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/ChildInfo.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@Getter
+public class ChildInfo {
+
+    private final String name;
+    private final String dateOfBirth;
+    private final String gender;
+    private final String relationshipToApplicant;
+    private final String relationshipToRespondent;
+    private final String countryOfResidence;
+
+    public ChildInfo(String name,
```

Comment:
Builder might give you a constructor for free. If not, you may want to use @AllArgsConstructor.

Reply:
:+1: done